wondrous-bot-admin/src/services/validators/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { TYPES } from "utils/constants";
+import * as Yup from "yup";
+
+export const ValidationError = Yup.ValidationError;
+
+const ALL_TYPES = [
+  TYPES.TEXT_FIELD,
+  TYPES.MULTI_QUIZ,
+  TYPES.NUMBER,
+  TYPES.ATTACHMENTS,
+  TYPES.SINGLE_QUIZ,
+  TYPES.LIKE_TWEET,
+  TYPES.FOLLOW_TWITTER,
+  TYPES.REPLY_TWEET,
+  TYPES.RETWEET,
+  TYPES.TWEET_WITH_PHRASE,
+  TYPES.SNAPSHOT_PROPOSAL_VOTE,
+  TYPES.SNAPSHOT_SPACE_VOTE,
+  TYPES.DISCORD_MESSAGE_IN_CHANNEL,
+  TYPES.JOIN_DISCORD_COMMUNITY_CALL,
+];
+
+const sharedValidation = {
+  type: Yup.string().required("Type is required").oneOf(ALL_TYPES, "Type is not valid"),
+  order: Yup.number().required("Order is required"),
+  prompt: Yup.string().required("Prompt is required"),
+};
+const stepTypes = {
+  [TYPES.TEXT_FIELD]: Yup.object().shape({
+    ...sharedValidation,
+  }),
+  [TYPES.MULTI_QUIZ]: Yup.object().shape({
+    ...sharedValidation,
+    options: Yup.array()
+      .of(
+        Yup.object().shape({
+          position: Yup.number().required("Position is required"),
+          text: Yup.string().required("Answer is required"),
+          correct: Yup.boolean().optional(),
+        })
+      )
+      .required("Options are required"),
+  }),
+  [TYPES.LIKE_TWEET]: Yup.object().shape({
+    ...sharedValidation,
```

Comment:
@Lamperoyge actually let's remove the prompt validation from the Twitter, Snapshot and Discord stuff I just realized we don't use prompts for these!